### PR TITLE
[DC-1274] Running Unit tests upgrade

### PR DESCRIPTION
**Pull request by jp3477**
Added quotes around shell variables in unit tests script, and updated README.

### CODE REVIEW

File: tests/run_tests.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ else
   cmd="tests/runner.py --test-path ${path} ${sdk_dir} --test-pattern $substring --coverage-file ${coverage_file}"
 fi
 
-(cd ${BASE_DIR}; PYTHONPATH=./:./data_steward:${PYTHONPATH} python ${cmd})
+(cd "${BASE_DIR}"; PYTHONPATH=./:./data_steward:${PYTHONPATH} python ${cmd})
```
Comment (nishanthpp93) [inline tests/run_tests.sh:62]: Do you mind adding quotes around `${PYTHONPATH}` and `${cmd}` as well?
```suggestion
(cd "${BASE_DIR}"; PYTHONPATH=./:./data_steward:"${PYTHONPATH}" python "${cmd}")
```

File: data_steward/tools/set_path.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ norm_path () {
 }
 
 DATA_STEWARD_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
-export DATA_STEWARD_DIR="$(norm_path ${DATA_STEWARD_DIR})"
+export DATA_STEWARD_DIR="$(norm_path "${DATA_STEWARD_DIR}")"
 
 # libs should appear first in PYTHONPATH so we can override versions from
 # the GAE SDK. (Specifically, we need oauth2client >= 4.0.0 and GAE uses 1.x.)
```
Comment (nishanthpp93) [inline data_steward/tools/set_path.sh:26]: nit: Quotes can be added to the `PYTHONPATH` too, but this is more for convention rather than actual use

**Review by nishanthpp93 — COMMENTED** [review]
Looks good, tests run fine. Added a minor suggestion.

**Review by nishanthpp93 — APPROVED** [review]

File: tests/run_tests.sh (hunk 1 above)
Comment (nishanthpp93) [reply tests/run_tests.sh:62]: This doesn't work because `cmd` is already a string, sorry about that

**Comment by jp3477** [comment]
@nishanthpp93 I closed this pull request and created an identical one for a branch in the upstream repo (not my fork).

**Comment by mark-velez** [comment]
@nishanthpp93 fyi I advised @jp3477 to re-issue the pull request using our typical developer workflow now that he has been added to all-of-us. I will just approve it since you approved this one.

**Pull request closed without merging**